components/members-list/index.js

```diff
@@ -4,18 +4,32 @@ import MemberListItem from 'components/member-list-item';
 import classNames from 'components/members-list/member-list.module.scss';
 import { membersContext } from 'store/members/members-context';
 
-const MembersList = () => {
+const MembersList = ({ searchTerm }) => {
   const {
     state: { membersArr }
   } = membersContext();
   if (membersArr) {
+    console.log(membersArr);
     return (
       <div className={classNames.container}>
-        {membersArr.map((ele) => (
-          <React.Fragment key={ele['id']}>
-            <MemberListItem memberDetails={ele} />
-          </React.Fragment>
-        ))}
+        {membersArr.length > 0 &&
+          membersArr
+            .filter((ele) => {
```

Comment:
same

Reply:
The same code is repeated twice and most of the if condition are returning the ele itself , could you look into having one util file and look for a way to make the function more concise